dsHdmiIn.c

```diff
@@ -0,0 +1,169 @@
+{
+    // no hdmi in, return not supported
+    return dsERR_OPERATION_NOT_SUPPORTED;
+}
+
+dsError_t dsHdmiInSelectZoomMode (dsVideoZoom_t eZoomMode)
+{
+    // no hdmi in, return not supported
+    return dsERR_OPERATION_NOT_SUPPORTED;
+}
+
+dsError_t dsHdmiInPauseAudio ()
+{
+    // no hdmi in, return not supported
+    return dsERR_OPERATION_NOT_SUPPORTED;
+}
+
+dsError_t dsHdmiInResumeAudio ()
+{
+    // no hdmi in, return not supported
+    return dsERR_OPERATION_NOT_SUPPORTED;
+}
+
+dsError_t dsHdmiInRegisterConnectCB (dsHdmiInConnectCB_t CBFunc)
+{
+    // no hdmi in, return not supported
+    return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsHdmiInGetCurrentVideoMode (dsVideoPortResolution_t *resolution)
+{
+
+    return dsERR_OPERATION_NOT_SUPPORTED;
+}
+
+dsError_t dsHdmiInRegisterSignalChangeCB (dsHdmiInSignalChangeCB_t CBFunc)
+{
+   return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsHdmiInRegisterStatusChangeCB (dsHdmiInStatusChangeCB_t CBFunc)
+{
+   return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsHdmiInRegisterVideoModeUpdateCB(dsHdmiInVideoModeUpdateCB_t CBFunc)
+{
+  return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsHdmiInRegisterAllmChangeCB (dsHdmiInAllmChangeCB_t CBFunc)
+{
+  return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsHdmiInRegisterAVLatencyChangeCB (dsAVLatencyChangeCB_t CBFunc)
+{
+ return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsHdmiInRegisterAviContentTypeChangeCB (dsHdmiInAviContentTypeChangeCB_t CBFunc)
+{
+  return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsIsHdmiARCPort (dsHdmiInPort_t iPort, bool *isArcPort)
+{
+  return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsGetEDIDBytesInfo (dsHdmiInPort_t iHdmiPort, unsigned char *edid, int *length)
+{
+  return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsGetHDMISPDInfo (dsHdmiInPort_t iHdmiPort, unsigned char *data)
+{
+  return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsSetEdidVersion (dsHdmiInPort_t iHdmiPort, tv_hdmi_edid_version_t iEdidVersion)
+{
+ return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsGetEdidVersion (dsHdmiInPort_t iHdmiPort, tv_hdmi_edid_version_t *iEdidVersion)
+{
+ return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsGetAllmStatus (dsHdmiInPort_t iHdmiPort, bool *allmStatus)
+{
+ return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsGetSupportedGameFeaturesList (dsSupportedGameFeatureList_t* features)
+{
+ return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsGetAVLatency (int *audio_latency, int *video_latency)
+{
+ return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsSetEdid2AllmSupport (dsHdmiInPort_t iHdmiPort, bool allmSupport)
+{
+ return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsGetEdid2AllmSupport (dsHdmiInPort_t iHdmiPort, bool *allmSupport)
+{
+ return dsERR_OPERATION_NOT_SUPPORTED;
+}
+dsError_t dsGetHdmiVersion(dsHdmiInPort_t iHdmiPort, dsHdmiMaxCapabilityVersion_t *maxCompatibilityVersion)
+{
+ return dsERR_OPERATION_NOT_SUPPORTED;
+}
+
+/** @} */
```

Comment:
Please remove these two (168 and 169) if not required.

Reply:
Done